mail-filter/mimedefang/mimedefang-2.84.ebuild

```diff
@@ -58,5 +61,9 @@ src_install() {
 	newinitd "${FILESDIR}"/${PN}.init ${PN}
 	newconfd "${FILESDIR}"/${PN}.conf ${PN}
 
-	dodoc -r examples
+	dodoc -r examples contrib
+}
+
+pkg_postinst() {
+	elog "You can install Mail::SpamAssassin (mail-filter/spamassassin) and HTML::Parser (dev-perl/HTML-Parser) even after installing if you require them as they are loaded at run-time."
```

Comment:
Wrap at 72 chars (of actual text output).

Reply:
see new commit